### ISSUE TITLE
The help file is in a difficult format to maintain and in German only

### ISSUE THREAD

**Issue by jdelange22** [daniel-lerch/vocup#55]
As the title says, the help file is in a .chm format, which is a kindof old format and not very easy to maintain. In fact the help file contents is HTML which itself is much easier to maintain and keep up-to-date. Would like to hear your thoughts about this. Perheps an online HTML version is an option?

**Comment by daniel-lerch** [comment]
I do not have time to update any help resources for Vocup so I just left the legacy `.chm` file to have at least something.

Feel free to bring up ideas for a new help if you would like to contribute to it. In terms of online help I also have a domain and server ready for Vocup to host anything that can run in a Docker container.

**Comment by jdelange22** [comment]
I understand this completely. I think that using an online help page is more according to today's standards and much easier to maintain. In fact you can even put a dedicated repository up for a website (idea) and documentation/help. I'm willing to assist in translation if this is needed.

**Comment by daniel-lerch** [comment]
An online help would definitely be desirable but my focus for Vocup is working on features if I ever get work on it. So I might be able to translate a website and online help into German and can discuss different technologies and take care of hosting. For creating a website and online help I do not have time.

**Comment by jdelange22** [comment]
Fair enough, I understand you want to focus on development on the program and not on maintaining a website. Might be useful to create a seperate issue regarding feature requests concerning a possible website, not cluttering the issue tracker of the main program. Perhaps in a seperate repo. Just a questing regarding the help-file in its current format. Do you happen to have the original help source or a ".hpp" file as this is needed for editing a .chm file as I read on the internet? If so, it is possible to make different files for each language.

**Comment by daniel-lerch** [comment]
Unfortunately, I do not have any of the original sources that Florian Amstutz used to compile the .chm file anymore.
